tools/navigation/ExitRoomFiles/exit_room_algo.cpp

```diff
@@ -0,0 +1,719 @@
+//exit-includes
+#include <iostream>
```

Comment:
If I remember correctly we talked about this code and you said it doesn't work well.

If this is doesn't work well please remove this code. We already have pretty stable exit room algorithm.